packages/itwinui-css/backstop/tests/input.html

```diff
@@ -502,5 +505,118 @@ <h2>No wrapping label</h2>
       </div>
 
     </section>
+
+    <br />
+
+    <h2>Search box</h2>
+    <section id="demo-search-box">
+
+      <h3>Current method</h3>
+      <div class="iui-input-container iui-inline-icon">
+        <input
+          placeholder="Placeholder…"
+          type="search"
+          class="iui-input"
+          id="search-1"
+        />
+        <svg-search
+          class="iui-input-icon"
+          aria-hidden="true"
+        />
+      </div>
+
+      <h3>Flex box</h3>
+      <div
+        class="iui-input-flex-container"
+        data-iui-size="small"
+      >
+        <span>
+          <svg-search
+            class="iui-input-icon"
+            aria-hidden="true"
+          /></span>
```

Comment:
nit: Can the closing tag be on a new line? Or maybe you are doing this to save a line?

Same for L549, L552, etc.

Reply:
this syntax is actually invalid. html (unlike jsx) doesn't have self-closing tags, so you need to explicitly write out `<svg-search></svg-search>` even if there is nothing inside.

https://stackoverflow.com/questions/3558119/are-non-void-self-closing-tags-valid-in-html5

Reply:
Should be repo wide search because we are using self-closing tags a lot. Other PR.

Reply:
you can still use the correct tags in new code. 😅

Reply:
you can still use the correct tags in new code. 😅